packages/puppeteer-core/src/bidi/core/Connection.ts

```diff
@@ -23,8 +23,8 @@ export type BidiEvents = {
  */
 export interface Connection<Events extends BidiEvents = BidiEvents>
   extends EventEmitter<Events> {
-  send<T extends keyof Commands>(
+  send<T extends keyof CommandMapping>(
```

Comment:
@Lightning00Blade do we need to rename from Commands to CommandMapping? `Commands` sound much better to me + it is a breaking change.

Reply:
Well not necessarily, I thought `CommandMapping` better represents what the type is.
And its harder to confuse with `Command` which is also an export.

As if it's a breaking change I am not sure, as I don't think the type is meant to be consumed outside Puppeteer API, if one wants to do that they should use the package directly.

Reply:
I meant that it is a breaking change for the webdriver-bidi-protocol package.